api/helpers/gamestate/moves/resolve-three/execute.js

```diff
@@ -0,0 +1,54 @@
+const GamePhase = require('../../../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Resolve three',
+
+  description: 'Returns new GameState resulting from requested resolve three move',
+
+  inputs: {
+    currentState: {
+      type: 'ref',
+      description: 'The latest GameState before the requesting player resolves a three card',
+      required: true,
+    },
+    /**
+     * @param { Object } requestedMove - Object describing the request to resolve a three
+     * @param { String } requestedMove.cardId - Card to retrieve from the scrap pile
+     * @param { MoveType.RESOLVING_THREE } requestedMove.moveType - Specifies that this a Resolving Three move
+     */
+    requestedMove: {
+      type: 'ref',
+      description: 'The move being requested. Specifies which player is asking to resolve a three',
+    },
+    playedBy: {
+      type: 'number',
+      description: 'Player number of player requesting move.',
+    },
+  },
+  sync: true, // synchronous helper
+  fn: ({ currentState, requestedMove, playedBy }, exits) => {
+    const { cardId } = requestedMove;
+    let result = _.cloneDeep(currentState);
+
+    const targetCard = result.scrap.find(card => card.id === requestedMove.cardId);
+    
+    result.scrap = result.scrap.filter(card => card.id !== cardId);
```

Comment:
This doesn't matter much, but I prefer to reduce the number of loops by using findIndex and then splice like 
```suggestion
    const targetCardIndex = result.scrap.findIndex(card => card.id === requestedMove.cardId);
    
    const [ targetCard ] = result.scrap.splice(targetCardIndex, 1);
```